website/docs/advanced-concepts/multiple-environments.md

```diff
@@ -0,0 +1,56 @@
+
+# Multiple Environments
+
+When building applications, there is usually a clear distinction between the version of the app that end users see and the version still under development/testing. Most software of reasonable complexity make this distinction clear by having different environments for development/testing and production use. This ensures that end users of the app aren't impacted by changes which are still in development. 
+
+Appsmith provides a way to define multiple environments for datasources. It allows you to switch between different environments while developing your app in edit mode. By switching environments, you are applying the datasource configurations defined in that environment.
+
+:::info
+The multiple environments feature is only available on [Business Edition](https://www.appsmith.com/pricing). 
+:::
+
+## Environments on Appsmith
+
+Appsmith provides two environments out of the box - **staging** and **production**. Staging environment configurations should ideally point to datasources which are used for development, QA or user acceptance testing. Production environment configurations should point to datasources which have live data that the end user see on the app.
```

Comment:
> that the end user see on the app
@harshilp24 @danciaclara there is a grammatical mistake in this statement. It should be "end user sees on the app"